### PR TITLE
Update python static testing

### PR DESCRIPTION
**Pull request by gchickma**
Migrate Python static testing from Alpine to Ubuntu and fix bugs

### CODE REVIEW

File: scripts/test/initialize-dependencies-static-python.sh [hunk 1]
```diff
@@ -8,6 +8,15 @@ ART_REPO_ID=$3
 ART_REPO_USER=$4
 ART_REPO_PASSWORD=$5
 
+if [ "$DEBUG" == "true" ]; then
+    echo "DEBUG - Script input variables..."
+		echo "BUILD_LANGUAGE_VERSION=$BUILD_LANGUAGE_VERSION"
```
Comment (timrbula) [inline scripts/test/initialize-dependencies-static-python.sh]: Could move this line back to be aligned w/ the other ones

File: Dockerfile-python-ubuntu [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+WORKDIR /opt/bin
+
+#Add Packages
+RUN apt-get -y update && apt-get --no-install-recommends -y install bash sed grep curl coreutils ca-certificates git jq make nodejs npm openssl openssh-client skopeo socat
+
+# Install img
+RUN curl -fSL "https://github.com/genuinetools/img/releases/download/v0.5.10/img-linux-amd64" -o "/opt/bin/img" && chmod a+x "/opt/bin/img"
+
+# Install nvm
```
Comment (timrbula) [inline Dockerfile-python-ubuntu]: Does `nvm` need to be installed? Are we switching node.js versions somewhere?

**Review by timrbula — CHANGES_REQUESTED** [review]
overall looks good to me as far as I can tell. Just a question and a minor suggestion

File: scripts/test/initialize-dependencies-static-python.sh (hunk 1 above)
Comment (gchickma) [reply scripts/test/initialize-dependencies-static-python.sh]: @timrbula Yep, done.

File: Dockerfile-python-ubuntu (hunk 2 above)
Comment (gchickma) [reply Dockerfile-python-ubuntu]: @timrbula Nope, I don't think so.  Good catch.  Removing.

**Pull request merged**